secure/storage/src/lib.rs

```diff
@@ -29,5 +32,21 @@ pub use crate::{
     vault::VaultStorage,
 };
 
+impl From<&SafetyRulesBackend> for Box<dyn Storage> {
```

Comment:
Could we not also update the safety_rules_manager.rs file to use this implementation? I think it currently does this by hand.

Reply:
yup! after our efforts merge, I don't want to make the merging more complicated than it might be...

Reply:
Sounds good!